routes.js

```diff
@@ -14,4 +14,74 @@ router.delete('/users',(req,res,next)=>{
         res.json(user);
     })
 })
-module.exports = router;
+
+      
+ //GET 
+
+ router.get('/leaderboard',(req,res,next)=>{
+    user.find({},(err,user)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.get('/companylist',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.get('/costumerdetail',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.get('/companydetail',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });  
+  });
+
+  router.get('/newslist',(req,res,next)=>{
+    user.find({},(err,news)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.post('/buy',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.post('/sell',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.post('/short',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.post('/cover',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.post('/takeloan',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });
+  });
+
+  router.post('/repayloan',(req,res,next)=>{
+    user.find({},(err,company)=>{
+        res.send("foobar");
+    });
+  });
+
```

Comment:
In all these routes, there is no need to query the database because you are as it is querying the user model everywhere which is wrong anyway. So, just do the `res.send(foobar)` for now. Also, there seems to be no need to pass the `next` middleware in the callback